src/configs/controllers/configController.ts

```diff
@@ -17,6 +23,42 @@ function configMapper(config: Config): components['schemas']['config'] {
   };
 }
 
+const sortFieldsMap = new Map<string, keyof Omit<Config, 'config'>>(
```

Comment:
Consider writing a type that represents all the sortable fields for better understanding
```suggestion
type SortableFields = keyof Omit<Config, 'config'>;
const sortFieldsMap = new Map<string, SortableFields>(
```

Reply:
changed